app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] Please change this method to current_user method that you created in the application_controller.rb file as per the requirement.

Reply:
Hey @shihabrafio, what the previous reviewer meant was to use the `current_user` method instead of finding the user using user_id from params. Like this: `@user = current_user`.